pkg/controller/windowsmachineconfig/tracker.go

```diff
@@ -105,7 +197,7 @@ func (t *Tracker) Reconcile() error {
 }
 
 // createStoreConfigMap creates the store ConfigMap without any data
```

Comment:
doc func name mismatch

Reply:
This should be fixed here. With things like doc mismatch I am fine with fixing them if you are making changes in the area.

Reply:
Please fix this

Reply:
The renaming is fixed in next commit.